virtwho/register.py

```diff
@@ -855,6 +888,57 @@ def associate_on_webui(self, hypervisor, guest):
                 return False
             return True
 
+    def subscription_on_webui(self, pool):
```

Comment:
new function to get on subscription details on webui by call api, which is used to test the virtual bonus pool comes from the correct hypervisor.